src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EntityParserTest.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EntityParserTest {
```

Comment:
Could we add some more scenarios when say payment section is missing or non scalable item is missing ?